### ISSUE TITLE
Unable to set container div id (v11)

### ISSUE THREAD

**Issue by sushantdhiman** [wbkd/react-flow#2510]
### Describe the Bug

With v10 you could set `id` for container element something like this

```tsx
 <ReactFlow
    id="custom_drawing"
    defaultEdges={edges}
    defaultNodes={nodes}
>
    <Controls />
    <Background />
</ReactFlow>

``` 

Which no longer works with `v11`, it renders element without any `id` attribute

```html
<div class="react-flow" data-testid="rf__wrapper" style="width: 100%; height: 100%; position: relative; overflow: hidden;"></div>
```

### Steps to Reproduce the Bug or Issue

Specify `id` for `<ReactFlow />`

### Expected behavior

I expected `id` attribute to be set in rendered element

### Platform

- OS: Linux
- Browser: Firefox
- Version: 105

**Comment by moklick** [comment]
That's a bug. Thanks for the report.

**Comment by moklick** [comment]
Fixed in v11.1.2
